backend/Zeus.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "API", "API", "{750068C9-CB1A-4988-B317-53BC1F0896BE}"

```

Comment:
Avoid full uppercase naming, even in case of acronym.
```suggestion
Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Api", "Api", "{750068C9-CB1A-4988-B317-53BC1F0896BE}"

```